### PR TITLE
fix(request-node): log config

### PR DESCRIPTION
**Pull request by benjlevesque**
The latest version of the node doesn't log anything due to a bug introduced in #1115

### CODE REVIEW

**Review by kevindavee — APPROVED** [review]

File: packages/request-node/src/config.ts [hunk 1]
```diff
@@ -190,19 +190,14 @@ export function getMnemonic(): string {
...
 }
 
-/**
- * Get log configs: level and mode, from command line argument, environment variables or default values.
- * logLevel is the maximum level of messages we will log
- * logMode defines the log format to display: `human` is a more readable log, `machine` is better for parsing
- *
- * @returns the log level
- */
-export function getLogConfig(): { logLevel: LogTypes.LogLevel; logMode: LogMode } {
```
Comment (benjlevesque) [inline packages/request-node/src/config.ts:200]: removing this beacuse I'm dropping it in my next PR, this is not the fix

File: packages/request-node/src/config.ts [hunk 2]
```diff
@@ -190,19 +190,14 @@ export function getMnemonic(): string {
...
-  return {
-    logLevel: getOption('logLevel', 'LOG_LEVEL', defaultValues.log.level),
-    logMode: getOption('logMode', 'LOG_MODE', defaultValues.log.mode),
-  };
-}
+/** logLevel is the maximum level of messages we will log */
+export const getLogLevel = (): LogTypes.LogLevel => {
+  const logLevelStr = getOption<keyof typeof LogTypes.LogLevel>('logLevel', 'LOG_LEVEL');
+  return LogTypes.LogLevel[logLevelStr] || defaultValues.log.level;
```
Comment (benjlevesque) [inline packages/request-node/src/config.ts:196]: this is the fix

**Comment by MantisClone** [comment]
The bug was that `LogLevel` is an enum backed by a number but the `LOG_LEVEL` env variable takes `DEBUG`, not `0`.

**Comment by MantisClone** [comment]
I deployed the Goerli Gateway and the logging is fixed. :tada: 
image

**Pull request merged**